src/tv2_offtube_showstyle/cues/OfftubeJingle.ts

```diff
@@ -35,27 +29,27 @@ export function OfftubeEvaluateJingle(
 	_rank?: number,
 	effekt?: boolean
 ) {
-	if (!config.showStyle.BreakerConfig) {
-		context.notifyUserWarning(`Jingles have not been configured`)
+	if (!context.config.showStyle.BreakerConfig) {
+		context.core.notifyUserWarning(`Jingles have not been configured`)
```

Comment:
If you should feel like resolving my other comment, it would also apply here. Again, this is not really important.